openedx/openedx-platform
# [WL-1240] [ENT-640] [ENT-628] [ENT-629] [ENT-630] [ENT-631] [ENT-634] [ENT-644] Bump to edx-enterprise to 0.48.1

The changes in this PR include:

1. Adds the custom Enterprise tagline to the top of the the Enterprise pages.
1. Add the `PLATFORM_DESCRIPTION` configuration variable for describing the platform; for now, to be used on the Enterprise landing pages.
1. Version bump edx-enterprise for the following changeset: https://github.com/edx/edx-enterprise/compare/0.46.8...0.48.1

The corresponding configuration PR is https://github.com/edx/configuration/pull/4083.

**JIRA tickets**: [ENT-628](https://openedx.atlassian.net/browse/ENT-628), [ENT-629](https://openedx.atlassian.net/browse/ENT-629), [ENT-630](https://openedx.atlassian.net/browse/ENT-630), [ENT-631](https://openedx.atlassian.net/browse/ENT-631), [ENT-634](https://openedx.atlassian.net/browse/ENT-634), [ENT-644](https://openedx.atlassian.net/browse/ENT-644), & [WL-1240](https://openedx.atlassian.net/browse/WL-1240).

**Testing instructions**:

To test the new variables portion of this PR:

1. Go to the programs enrollment landing page. Notice the tagline you entered being in the header next to the logo. Notice also the platform description on the sidebar.
1. Go to the data sharing consent page. Notice the tagline you entered being in the header next to the logo.
1. Go to the course enrollment landing page. Notice the tagline you entered being in the header next to the logo.

**Reviewers**
- [ ] @haikuginger @itsjeyd 
- [ ] @douglashall @brittneyexline @tuchfarber 

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
  ENTERPRISE_TAGLINE: "Why, hello there."
  PLATFORM_DESCRIPTION: "What a fancy description!"
```
